doc/Makefile.am

```diff
@@ -31,6 +31,9 @@ SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
 
+dist_bin_SCRIPTS = mkapiref.py
+dist_doc_DATA = source/index.rst source/programmers-guide.rst
```

Comment:
ditto.  If these files are just needed in an archive, add them to EXTRA_DIST.

Reply:
Hmm, while index.rst is not much useful itself, programers-guide.rst might be a good part of devel package. But I think better would be if man page were generated also. That would be a bit better consumable from terminal.

Reply:
I have no plan to add generated manual pages to release archive at the moment.
People can access nghttp2.org/ngtcp2 to get html version of manual, or can git clone the repo.

Reply:
I did not propose adding generated man pages. I do not think they should be part of source archive, no. But there should be a make target to create them in a simple way.